cmd/kubeadm/app/phases/upgrade/postupgrade.go

```diff
@@ -130,12 +141,15 @@ func removeOldDNSDeploymentIfAnotherDNSIsUsed(cfg *kubeadmapi.ClusterConfigurati
 			}
 		}
 
-		// We don't want to wait for the DNS deployment above to become ready when dryrunning (as it never will)
-		// but here we should execute the DELETE command against the dryrun clientset, as it will only be logged
-		err := apiclient.DeleteDeploymentForeground(client, metav1.NamespaceSystem, deploymentToDelete)
-		if err != nil && !apierrors.IsNotFound(err) {
-			return err
+		for _, deletedDeployment := range deletedDeployments {
```

Comment:
`deletedDeployment, _`?

Reply:
`_, deletedDeployment` is fine in term of ordering IMO

Reply:
derp, let me delete this comment.